### PR TITLE
ci-images-mirror: reload config file

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform

### CODE REVIEW

File: cmd/ci-images-mirror/main.go [hunk 1]
```diff
@@ -86,6 +89,27 @@ func newOpts() *options {
...
 }
 
+func (o *options) loadConfig(caller string) error {
+	o.configMutex.Lock()
+	defer o.configMutex.Unlock()
+	now := time.Now()
+	if o.configFile == "" {
+		return fmt.Errorf("cannot load config when the config file is empty")
+	}
+	if o.config != nil && now.Before(o.configLastModifiedAt.Add(30*time.Minute)) {
```
Comment (deepsm007) [inline cmd/ci-images-mirror/main.go:99]: Any particular reason for checking before 30 min specifically?

Comment (hongkailiu) [reply cmd/ci-images-mirror/main.go:99]: I think anything that is smaller than `1h` is good enough.
`1h` is the interval to run the sync on ci-sup and art images.

**Comment by hongkailiu** [comment]
/hold

Will watch the rolling tomorrow morning.

**Comment by hongkailiu** [comment]
Screenshot 2024-06-14 at 10 41 28 AM

Seems working to me.

**Pull request merged**